packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol

```diff
@@ -53,36 +53,26 @@ library AssetLogic {
 
   /**
    * @notice Handles transferring funds from msg.sender to the Connext contract.
-   * @dev If using the native asset, will automatically wrap
+   * @dev Only allows native asset to be used for fee
```

Comment:
comment about `@param _fee` not entirely accurate, all of native asset is meant for for

Reply:
@LayneHaber can we remove the `_fee` param here entirely? as well as the check below?